pyproject.toml

```diff
@@ -1,7 +1,15 @@
 
 [tool.black]
 target-version = ['py38']
-include = '.*/backend-tests/.*\.py$'
+include = '''
+(
+  /(
+      testutils/.*\.py$
+    | backend-tests/.*\.py$
+    | tests/.*\.py$
+  )/
+)
+'''
```

Comment:
@oleorhagen Please pay close attention to this regex. I have low confidence in that I got it right

Reply:
Not completely right I am afraid, so good instincts :)
Try this:
```python
include = '''
(
  /(
      backend-tests
    | testutils
    | tests
  )/.*\.py$
)
```

Reply:
That works, thanks!